### DISCUSSION TITLE
History went missing

### DISCUSSION THREAD

**Discussion by klkohl** [PeteRager/lennoxs30#326] [General]
Just yesterday I noticed that my entity history prior to May 5 at 4 pm exactly is missing for this integration. History for other integrations appears to still be there. Any idea what might be going on?

**Comment by PeteRager** [comment]
Look at your configuration for the recorder

https://www.home-assistant.io/integrations/recorder/

**Comment by klkohl** [comment]
I'm just using default_config:.

**Comment by PeteRager** [comment]
Please read that documentation.  The default retention period is 10 days.  You'll need to change the recorder configuration if you need more.

**Comment by klkohl** [comment]
I understand that default retention is 10 days and realized I may have misspoke. It's the long term statistics that are missing.  I'm attaching a screenshot that may help show the issue. It's a graph of my S40 temperature reading. I'm also showing an outside thermomter that doesn't have the problem. 
![image](https://github.com/PeteRager/lennoxs30/assets/88909821/65ed59dc-40ef-4546-934b-071a10a20d5c)

**Comment by klkohl** [comment]
Never mind. I think I remember what happened. Sorry to bother you.
